geoips/filenames/base_paths.py

```diff
@@ -127,6 +128,10 @@ def initialize_paths():
         "GEOIPS_COPYRIGHT": "NRL-Monterey",
         "GEOIPS_COPYRIGHT_ABBREVIATED": "NRLMRY",
         # Configuration and Queue
+        # Note: platformdirs provides functions to retrieve appropriate directory paths
+        # for different use types on different operating systems. GEOIPS_CACHE_DIR will
+        # resolve to a different path on Linux/WSL, OSX, and Windows.
+        "GEOIPS_CACHE_DIR": platformdirs.user_cache_dir("geoips"),
```

Comment:
Note I have been collecting various environment variables that are required for dependency cache locations here:

https://github.com/NRLMMD-GEOIPS/geoips/blob/328f7d160d5e4953d7f129b751b6f2f310123043/setup/config_geoips#L40

![image](https://github.com/user-attachments/assets/97775c1a-5cd1-40e8-803d-e848323bdfa4)

Would it be worthwhile to document these along with GEOIPS_CACHE_DIR?  I think GEOIPS_CACHE_DIR is totally separate (and will be overriden with GEOIPS_CACHE_DIR if it is set in the environment, correct?  And later geoips config file when we have it?), but might be nice to keep it all in one place?

Reply:
^ I think so! To be specific, I think it'd be helpful to document these alongside of GEOIPS_CACHE_DIR. And yes, _thankfully_ this will all be taken over by the config file